### DISCUSSION TITLE
Support for multiple currency

### DISCUSSION THREAD

**Discussion by TheKalpit** [solidtime-io/solidtime#241] [Feature requests]
Would be great to assign a currency to a client/project which is different from organization's currency.

**Comment by korridor** [comment]
We decided to move feature requests into github discussions, to separate feature requests from bugs and allow for better prioritization. If this feature is important to you, you can upvote it to let us know.

**Comment by excalith** [comment]
That would be great to assign different currencies for clients/projects and makes working worldwide easy, especially prevents recalculations (ie. rate per hour in a different currency) while creating an invoice.

**Comment by smileBeda** [comment]
This is almost a must have to capture what Clockify has (and I notice SolidTime offers imports from Clockify, so it appears it at least considers itself some sort of "similar to" tool)

It would be great to have this - it does not matter much if on organization level, client level or member level!

**Comment by smncd** [comment]
Chiming in here too, it'd be brilliant to be able to select currency per project
